### PR TITLE
Adding loader for (Compmusic) Carnatic Music Rhythm

### PR DESCRIPTION
**Pull request by genisplaja**
### Description

Please include the following information at the top level docstring for the dataset's module mydataset.py:

- [x] Describe annotations included in the dataset
- [x] Indicate the size of the datasets (e.g. number files and duration, hours)
- [x] Mention the origin of the dataset (e.g. creator, institution)
- [x] Describe the type of music included in the dataset
- [x] Indicate any relevant papers related to the dataset
- [x] Include a description about how the data can be accessed and the license it uses (if applicable)

#### Dataset loaders checklist:

- [x] Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file.
- [x] Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`.
- [x] Create a module in mirdata, e.g. `mirdata/my_dataset.py`
- [x] Create tests for your loader in `tests/datasets/`, e.g. `test_my_dataset.py`
- [x] Add your module to `docs/source/mirdata.rst` and `docs/source/table.rst`
- [x] Run `tests/test_full_dataset.py` on your dataset.

### CODE REVIEW

**Comment by genisplaja** [comment]
Hi @nkundiushuti, @magdalenafuentes and @harshpalan. As discussed the last week, I got this dataloader done using the optional dependencies as usual. Do you have any idea why the docs are not rendering properly? I went through the raw but I cannot find any specific error there. Regarding the rest of TODOS, looks like it is ready to be reviewed :)

File: docs/source/table.rst [hunk 1]
```diff
@@ -53,6 +53,19 @@
...
      - :cante:`\ `
 
+   * - .. line-block::
+
+        (CompMusic)
+        Carnatic Rhythm
+
+     - - audio: 🔑
+       - annotations: 🔑
+     - - :ref:`beats`
```
Comment (nkundiushuti) [inline docs/source/table.rst:63]: shouldn here be talas instead of meter?

File: mirdata/datasets/compmusic_carnatic_rhythm.py [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+import numpy as np
+
+from mirdata import annotations, core, io, jams_utils
+from smart_open import open
+
+
+try:
+    from openpyxl import load_workbook as get_xlxs
+except ImportError:
+    logging.error(
```
Comment (nkundiushuti) [inline mirdata/datasets/compmusic_carnatic_rhythm.py:62]: it looks like this is not covered by test? is this to be compliant with what Harsh is doing on partial dependency checks? maybe we could have a global test for that

File: mirdata/datasets/compmusic_carnatic_rhythm.py [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+    Returns:
+        BeatData: beat annotations
+
+    """
+    beat_times = []
+    beat_positions = []
+
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        beat_times.append(float(line[0]))
```
Comment (nkundiushuti) [inline mirdata/datasets/compmusic_carnatic_rhythm.py:261]: doesn't this data have the sam marked? then it should be like beats and downbeats? if no sam is marked then you can have it as beats

File: mirdata/datasets/compmusic_carnatic_rhythm.py [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+    """Load meter
+
+    Args:
+        fhandle (str or file-like): Local path where the meter annotation is stored.
+
+    Returns:
+        float: meter annotation
+
+    """
+    reader = csv.reader(fhandle, delimiter=",")
```
Comment (nkundiushuti) [inline mirdata/datasets/compmusic_carnatic_rhythm.py:283]: this basically gives you the tala of the piece

File: mirdata/datasets/compmusic_carnatic_rhythm.py [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+            audio_path=self.audio_path,
+            beat_data=[(self.beats, "beats")],
+            metadata={
+                "meter": self.meter,
+                "mbid": self.mbid,
+                "name": self.name,
+                "artist": self.artist,
+                "release": self.release,
+                "lead_instrument_code": self.lead_instrument_code,
+                "taala": self.taala,
```
Comment (nkundiushuti) [inline mirdata/datasets/compmusic_carnatic_rhythm.py:220]: so it has the taala annotated, then what is the meter annotation?

**Review by nkundiushuti — APPROVED** [review]

**Pull request merged**